frontend/src/TooljetDatabase/Table/index.jsx

```diff
@@ -160,6 +182,55 @@ const Table = ({ openCreateRowDrawer, openCreateColumnDrawer }) => {
     }
   );
 
+  const columHeaderLength = useMemo(() => headerGroups[0]?.headers?.length || 0, [headerGroups]);
+
+  const moveColumnCreateElement = useCallback(() => {
+    setTimeout(() => {
+      const initialxAxis = columnCreatorElement.current?.getBoundingClientRect().x;
+      const fullWidth = window.innerWidth;
+      setWidth((prevState) => ({
+        ...prevState,
+        screenWidth: fullWidth,
+        xAxis: initialxAxis,
+      }));
+    }, 200);
+  }, [setWidth, columnCreatorElement]);
+
+  useEffect(() => {
+    moveColumnCreateElement();
+  }, []);
+
+  useEffect(() => {
+    moveColumnCreateElement();
+  }, [columHeaderLength]);
+
+  const widthOfScreen = width.screenWidth > 0 ? width.screenWidth : wholeScreenWidth;
+
+  const colStyles = {
+    position: 'absolute',
+    top: '0px',
+    right: '-28px',
+    width: '29px',
+    height: '30px',
+    borderWidth: '0px 1px 1px 1px',
+    borderStyle: 'solid',
+    borderRadius: '0px',
+  };
+
+  const colStyles1 = {
+    position: 'fixed',
+    top: '112px',
+    right: '0px',
+    width: '29px',
+    height: '30px',
+    borderWidth: '0px 1px 1px 1px',
+    borderStyle: 'solid',
+    borderRadius: '0px',
+    zIndex: '1',
+  };
```

Comment:
Can we move this to styles.scss?
Also, could you check if these can be made responsive as well?